aliases-private.example.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+alias dcea="docker compose exec django-dev"  # docker compose exec app
+alias prod="open 'https://ninagen.nina.no/'"
+alias local="open 'localhost:8000'"
```

Comment:
`open` is not a valid command on Linux. I would suggest keeping `xdg-open` as default. It might be possible to set `open` if detected.

```bash
open_command="xdg-open"
if ! type "open" > /dev/null; then
  open="open"
fi
```

Reply:
Ait, removed the file